File: backEnd/lunch.BusinessLogic/Security/ExternalUserDetails.cs
using lunch.Domain.Security;

namespace lunch.BusinessLogic.Security
{
    public class ExternalUserDetails
    {
        public UserType UserType { get; set; }

        public string Id { get; set; }

        public string Email { get; set; }

        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string DisplayName { get; set; }

        public string PictureUrl { get; set; }
    }
}

File: backEnd-go/businessLogic/security/externalUserDetails.go
package security

import "github.com/bcrusu/lunch/backEnd-go/domain/security"

type ExternalUserDetails struct {
	UserType    security.UserType
	ID          string
	Email       string
	FirstName   string
	LastName    string
	DisplayName string
	Description string
	PictureURL  string
}
